src/test/runner.ts

```diff
@@ -93,17 +100,25 @@ export class Runner {
       await this._flushOutput();
       return 'failed';
     }
-    if (result === 'forbid-only') {
+    if (result?.status === 'forbid-only') {
       console.error('=====================================');
       console.error(' --forbid-only found a focused test.');
+      for (const testName of result?.testNames)
+        console.error(` -- ${testName}`);
```

Comment:
I'd like to see full title like the reporter does: `dir/a.spec.ts:55 > suite subsuite spec`